### DISCUSSION TITLE
只显示评论个数，但没有评论内容😕

### DISCUSSION THREAD

**Discussion by PoplarTang** [walinejs/waline#1173] [Q&A] [answered]
查看请求数据，能看到数量是对的，在vercel.app里也能看到评论内容，但是前端就加载不到数据：

![image](https://user-images.githubusercontent.com/4375208/177531528-012c89a2-f896-4a7d-9d75-3cbff1d5396d.png)

**Comment by lizheming** [comment, marked as answer]
看下数据的状态是不是审核中，或者是子评论，父评论被删除了？

**Reply by PoplarTang** [reply]
都是默认已通过状态，没有父评论哦，刚建站，都是自己创建的默认根级评论。
![image](https://user-images.githubusercontent.com/4375208/177570236-8edc210a-beda-41eb-9dea-0a9c1a0ff5a2.png)

**Reply by PoplarTang** [reply]
我的博客地址是这个：http://tangyang.studio/
另外我参考的这个博主的评论系统好像也加载不出来了：https://www.haomwei.com/technology/from-Disqus-to-Waline.html

**Reply by lizheming** [reply]
是不是 LeanCloud 的数据库有复用，全新建一个数据库试试呢。屠城的是另外一个问题，已经在和他跟进了。

**Reply by PoplarTang** [reply]
这么晚还没休息啊，辛苦怡红大神了，我新加了个数据库，重新部署了个vercel应用。
通过GET请求还是拿不到数据，只有个数：https://tangyang.vercel.app/comment?path=%2F2022%2F07%2Fblog-hexo.html&pageSize=&page=1&lang=zh-CN
但是可以看到提交评论时候POST请求是能正常返回评论内容的。

![image](https://user-images.githubusercontent.com/4375208/177590080-b79e7209-e2b2-4d02-9751-0555a553b7b4.png)

**Reply by lizheming** [reply]
我明天自己操作下试试吧，如果不介意的话可以邮件发我一下你的Leancloud的key和secret相关的配置我试一下

**Reply by PoplarTang** [reply]
找到原因了，谢了怡红公子，屠城提供了pageSize参数，但是我没有设置值，也就是说这个GET请求的服务器认为pageSize是0，
`https://tangyang.vercel.app/comment?path=%2F2022%2F07%2Fblog-hexo.html&pageSize=&page=1&lang=zh-CN`
我加上这个参数值pageSize=20就可以了。
